src/solana/SolanaEngine.ts

```diff
@@ -502,7 +503,8 @@ export class SolanaEngine extends CurrencyEngine<
   // // Public methods
   // // ****************************************************************************
 
-  async startEngine(): Promise<void> {
+  async startEngine(opts?: StartEngineOpts): Promise<void> {
+    const { lightMode = false } = opts ?? {}
```

Comment:
Since we are moving `lightMode` to the `makeCurrencyEngine` call, this will become `this.lightMode` instead of being an option.